test/functional/rpc_packages.py

```diff
@@ -29,6 +29,7 @@ class RPCPackagesTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
+        self.extra_args = [["-whitelist=127.0.0.1"]]
```

Comment:
nit: Could only add the needed permission flag? and add a comment?

Reply:
Done